### ISSUE TITLE
Expose more API properties

### ISSUE THREAD

**Issue by chrisg32** [chrisg32/AvaloniaPdfViewer#2]
- [ ] Show sidebar (two way binding) (DirectProperty)
- [ ] Show sidebar button (StyledProperty)
- [ ] Show zoom controls (StyledProperty)
- [ ] Show page controls (StyledProperty)
- [ ] Page (two way binding) (DirectProperty)
- [ ] Zoom (two way binding) (DirectProperty)

**Comment by lmponcio** [comment]
Hi @chrisg32, thank you for this project.

Would it be ok if I send a small PR with this part of the issue addressed?

>  Page (two way binding) (DirectProperty)

If you have specific guidelines for contrubuting please let me know.

Thanks again.

**Comment by chrisg32** [comment]
@lmponcio Yes, please do. I can't promise a fast review but I will try to get to it within the week.
